llvm.proj

```diff
@@ -70,7 +71,7 @@
     <_LLVMBuildArgs Condition="'$(BuildOS)' == 'Windows_NT'" Include='-DLLVM_USE_CRT_RELEASE=MT' />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetArchitecture)' != 'arm' and '$(TargetArchitecture)' != 'arm64'">
+  <ItemGroup Condition="'$(BuildOS)' == 'OSX' or ('$(TargetArchitecture)' != 'arm' and '$(TargetArchitecture)' != 'arm64')">
```

Comment:
In principle we can also build linux-arm64,linux-arm,windows-arm64,windows-arm libclang binaries, but for now I just allow it for osx-arm64, only.  We could probably just build all of them.